### ISSUE TITLE
Ask Me Anything session with a Kaggle Grandmaster Vladimir I. Iglovikov

### ISSUE THREAD

**Issue by khuyentran1401** [khuyentran1401/machine-learning-articles#29]
## TL;DR
Data science practice of high achiever in data science

### Article Link
https://towardsdatascience.com/ask-me-anything-session-with-a-kaggle-grandmaster-vladimir-i-iglovikov-942ad6a06acd

### Author
Vladimir Iglovikov

## Key Takeaways
*  Use minimal Jupyter Notebook. Write code base that can be refactored in the future. This will speed you up later on
*  Write unit tests wherever possible
*  Minimize the use of a mouse (write down hotkeys in a piece of paper)
*  Do not use social networks
*  Check email only a few times a day
*  Skills in Kaggle are just the small subset of real skills in the industry
*  Invest on good GPU can be a good investment over a long term
*  Experiment with the parameters without the certainty why the parameters may work
* To get above the level of experimenting, you need to educate yourself with books, courses, write better code. Learn and experiment at the same time
* Read what others do and try to understand what they do. The next time you could do the same techniques with similar problems

## Useful Tools
*  PyCharm 
*  DVC: Version control
* Trellow: Keep track of the tasks

## Comments/ Questions
* Really useful advice not just for Kaggle but for data science practice in general
